packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -4046,6 +4046,36 @@ addEventListener('fetch', event => {});`
 			expect(std.err).toMatchInlineSnapshot(`""`);
 		});
 
+		it("should fail to deploy to the workers.dev domain if email is unverified", async () => {
+			writeWranglerToml({ workers_dev: true });
+			writeWorkerSource();
+			mockUploadWorkerRequest({ available_on_subdomain: false });
+			mockSubDomainRequest();
+			msw.use(
+				rest.post(
+					`*/accounts/:accountId/workers/scripts/:scriptName/subdomain`,
+					async (req, res, ctx) => {
+						return res.once(
+							ctx.json(
+								createFetchResult(null, /* success */ false, [
+									{
+										code: 10034,
+										message: "workers.api.error.email_verification_required",
+									},
+								])
+							)
+						);
+					}
+				)
+			);
+
+			await expect(
+				runWrangler("deploy ./index")
+			).rejects.toThrowErrorMatchingInlineSnapshot(
+				`"Please verify your account's email address and try again."`
```

Comment:
Should this snapshot contain the 2nd line? 	"Check your email for a verification link, or login to https://dash.cloudflare.com and request a new one."

Reply:
The additional text is added as a "note" to the error. This error snapshot matcher only considers the "message" of the error.

Reply:
If we care about the second line then I think we would need to catch the error and then stringify it before checking it against a snapshot.